test/forms/OptionalSupportingDocsFormProviderSpec.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import base.SpecBase
+
+class OptionalSupportingDocsFormProviderSpec extends SpecBase {
+
+  "Binding a form with invalid data" when {
+    val data = Map("" -> "")
+    val form = new OptionalSupportingDocsFormProvider()().bind(data)
+
+    "Invalid data present" should {
+
+      "result in a form with errors" in {
+        form.hasErrors mustBe true
+      }
+    }
+
+  }
```

Comment:
Needs a test for the successful binding case

Reply:
couldn't get it to work tbh and forgot to ask about it